moped-editor/src/views/projects/signalProjectTable/RenderSignalLink.js

```diff
@@ -5,7 +5,7 @@ const RenderSignalLink = ({ signals }) => {
   return (
     <span>
       {signals.map((signal, index) => (
-        <React.Fragment key={signal.signal_id}>
+        <React.Fragment key={signal.component_id}>
```

Comment:
Reminds me that I made a hacky workaround in another branch. 🙃 I'm going to remove that, and I'll pull in this work when it gets merged. I don't see the error come up, but I do think that `feature_signals.id` is guaranteed to be unique. I was trying to refresh myself on the columns and found some records that share `component_id` but their soft delete status prevents some from returning from the query. Could a double-check on that, though.

<img width="1336" alt="Screenshot 2023-09-29 at 1 19 11 PM" src="https://github.com/cityofaustin/atd-moped/assets/37249039/ac2dff8c-f310-4788-8be5-0a963511a123">

Reply:
oh good call, ill switch to that instead of component id